### PR TITLE
Warn users when session cookies are split

### PR DESCRIPTION
**Pull request by JoelSpeed**
## Description

Adds a warning to users when sessions are being split, recommending them to use a server side session storage implementation such as redis

## Motivation and Context

To start potential changes inferred by #482 

## How Has This Been Tested?

Not tested but a minor change that shouldn't need much in the way of testing

## Checklist:

- [x] My change requires a change to the documentation or CHANGELOG.
- [x] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: pkg/sessions/cookie/session_store.go [hunk 1]
```diff
@@ -129,6 +130,7 @@ func NewCookieSessionStore(opts *options.SessionOptions, cookieOpts *options.Coo
 // it into a slice of cookies which fit within the 4kb cookie limit indexing
 // the cookies from 0
 func splitCookie(c *http.Cookie) []*http.Cookie {
+	logger.Printf("WARNING: Sessions exceeds 4kb cookie limit. Multiple cookies are required for this session. Please use server side session storage (eg. Redis).")
```
Comment (steakunderscore) [inline pkg/sessions/cookie/session_store.go:133]: How do you feel about change `WARNING` to `DEPRECATED`?

Comment (JoelSpeed) [reply pkg/sessions/cookie/session_store.go:133]: My thinking was that we haven't officially decided to deprecate yet, so shouldn't say it's deprecated? I was thinking we would add deprecated once we have gathered more feedback, though, changing it now, might make it easier to get feedback? 🤔

Comment (syscll) [inline pkg/sessions/cookie/session_store.go:133]: I think something like this reads better:

```suggestion
	logger.Printf("WARNING: Multiple cookies are required for this session as it exceeds the 4kb cookie limit. Please use server side session storage (eg. Redis) instead")
```

**Comment by JoelSpeed** [comment]
@syscll Applied your suggestion

**Pull request merged**